packages/registry/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "loom-registry",
+  "version": "0.0.1",
+  "description": "Centralised store for loom modules",
+  "main": "lib/index.js",
+  "repository": "https://github.com/loomjs/loom/tree/master/packages/loom-registry",
+  "author": "Alexandr Lazarev <diedsmiling@gmail.com>, Omar Ahmed <omar@omar.de>",
```

Comment:
Same as above, I vote for using the more generic Loom Contributors as the author